pkg/contracts/solana/gateway_message.go

```diff
@@ -14,6 +14,7 @@ const (
 	InstructionWhitelistSplToken byte = 3
 	InstructionExecute           byte = 5
 	InstructionExecuteSPL        byte = 6
+	InstructionIncrementNonce    byte = 7
 )
 
 var InstructionIdentifier = []byte("ZETACHAIN")
```

Comment:
it would be good to add a comment and maybe a code to the withdraw function in the program; it would make it easier to understand why this specific format is used for creating the hash